test/php/gutenberg-coding-standards/Gutenberg/Sniffs/Commenting/SinceTagSniff.php

```diff
@@ -442,108 +441,77 @@ protected function check_below_minimum_visibility( $visibility ) {
 	 *
 	 * @param File $phpcs_file    The file being scanned.
 	 * @param int  $stack_pointer The position to start looking for the docblock.
-	 * @return array|false An associative array containing the start and end tokens of the docblock, or false if not found.
+	 * @return int|false The last token on the previous line.
 	 */
-	protected static function find_hook_docblock( File $phpcs_file, $stack_pointer ) {
+	protected static function find_previous_line_token( File $phpcs_file, $stack_pointer ) {
```

Comment:
I believe the method description in the docblock needs to be updated to reflect the changes made to this method.

Reply:
Well spotted.
Fixed in cff54f744d890d095ec5079616d27afb61fdd02a.